pkgs/development/libraries/isl/0.20.0.nix

```
@@ -1,29 +1,16 @@
-{ lib, stdenv, fetchurl, gmp }:
-
-stdenv.mkDerivation rec {
-  name = "isl-0.20";
-
-  src = fetchurl {
-    urls = [
-      "mirror://sourceforge/libisl/${name}.tar.xz"
-      "https://libisl.sourceforge.io/${name}.tar.xz"
-    ];
-
-    sha256 = "1akpgq0rbqbah5517blg2zlnfvjxfcl9cjrfc75nbcx5p2gnlnd5";
-  };
-
-  buildInputs = [ gmp ];
+{ callPackage, lib, stdenv, fetchurl, gmp }:
 
+callPackage ./generic.nix rec {
+  version = "0.20";
+  urls = [
+    "mirror://sourceforge/libisl/libisl-${version}.tar.xz"
+    "https://libisl.sourceforge.io/libisl-${version}.tar.xz"
```

```suggestion
    "mirror://sourceforge/libisl/isl-${version}.tar.xz"
    "https://libisl.sourceforge.io/isl-${version}.tar.xz"
```

same goes for the others, I think